packages/dev/src/plugin.ts

```diff
@@ -69,7 +69,9 @@ export class DevtoolsPlugin implements ISender {
   protected pending: Record<string, ResolveRejctPromise> = {};
   protected pages: Array<Page> = [];
 
-  constructor() {
+  private readonly customPort?: number;
```

Comment:
NIT: put this in an object (DevToolsPluginOptions). This way future additions will just be another property in this object, and we won't risk having to ship breaking changes.